corehq/apps/hqmedia/view_helpers.py

```diff
@@ -114,18 +115,42 @@ def update_multimedia_paths(app, paths):
 
 def download_audio_translator_files(domain, app, lang):
     # Get bulk app translation single sheet data
+    headers = get_bulk_app_sheet_headers(app, single_sheet=True, lang=lang, eligible_for_transifex_only=True)
+    headers = headers[0]    # There's only one row since these are the headers for the single-sheet format
+    headers = headers[1]    # Drop the first element (sheet name), leaving the second (list of header names)
+    text_index = headers.index('default_' + lang)
+    audio_index = headers.index('audio_' + lang)
     sheets = get_bulk_app_single_sheet_by_name(app, lang, eligible_for_transifex_only=True)
-    translations = sheets[SINGLE_SHEET_NAME]
-
-    # Get missing multimedia data
-    missing_media_rows = download_multimedia_paths_rows(app, only_missing=True)
-    missing_media_paths = [row[0] for (sheet_name, row) in missing_media_rows]
+    audio_rows = [row for row in sheets[SINGLE_SHEET_NAME] if row[audio_index]]
+
+    # Create dict of rows, keyed by label text for de-duplication
+    rows_by_text = defaultdict(list)
+    for row in audio_rows:
+        rows_by_text[row[text_index]].append(row)
+
+    # Create file for translator
+    translator_workbook = openpyxl.Workbook()
+    sheet0 = translator_workbook.worksheets[0]
+    sheet0.title = "filepaths"
+    sheet0.append([lang, "audio"])
+    sheet1 = translator_workbook.create_sheet("verif")
```

Comment:
verif?

Reply:
That's what the original script does, but agreed it's not ideal. Updated in c720a97810a

Reply:
i think they just misspelt it there itself.